fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -146,6 +159,43 @@ def load_data(self, X, y, **kwargs):
         super().load_data(X, y, event=_ALL, **kwargs)
 
 
+class EqualOpportunity(ConditionalSelectionRate):
+    r"""Implementation of Equal Opportunity as a moment.
+
+    Adds conditioning on `y=1` compared to Demographic parity, i.e.
+
+    .. math::
+       P[h(X) = 1 | A = a, Y = 1] = P[h(X) = 1 | Y = 1] \; \forall a, y,
+
+    This implementation of :class:`ConditionalSelectionRate` defines
+    events corresponding to the unique values of the `Y` array.
+
+    The `prob_event` :class:`pandas:pandas.DataFrame` will record the
+    fraction of the samples corresponding to each unique value of `y = 1` in
+    the `Y` array.
+
+    The `index` MultiIndex will have a number of entries equal to
+    the number of unique values for the sensitive feature, multiplied by
+    the number of unique values of the `Y` array, multiplied by two (for
+    the Lagrange multipliers for positive and negative constraints).
+
+    With these definitions, the :meth:`signed_weights` method
+    will calculate the costs like how it is calculated in Example 4 of
```

Comment:
Minor thing but how about "will calculate the costs similarly to Example 4 ….. but only for all `y=1` samples "

Reply:
Yes, I have written that in line number 186 below. I can rephrase it to this though!